cluster-sim/cluster_sim/utils.py

```diff
@@ -88,26 +88,23 @@ class Persisted(object):
     filename = None
     default_state = {}
```

Comment:
Are any of the changes below this are required for this patch.

for server in self.servers.values():
   server.stop_corosync()
   server.stop_pacemaker()
   server.save()

Above is all that is required isnt it.

Adding in other changes just leads to unexpected changes in behaviour.